### PR TITLE
Krn ticket28 application background

### LINKED ISSUE #28: Application Background Color
As the application user, I get tired of boring black backgrounds, so I would like to have a more enjoyable background color.

As a non-technical person, however, I don't know what my options are. I would appreciate the team giving me some choices.

### PR DESCRIPTION
**Pull request by karlarn**
# Description
When the console first runs you're prompted with a greeting and to choose the background of the console.
closes #28 
## Type of change
Please delete options that are not relevant.
- [X ] New feature (non-breaking change which adds functionality)
# Testing Instructions
Run the Program and choose your background color.
# Checklist:
- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by jandershock — APPROVED** [review]
:shipit:

**Review by joshdbarton — CHANGES_REQUESTED** [review]

File: TabloidCLI/Program.cs [hunk 1]
```diff
@@ -12,10 +12,49 @@ static void Main(string[] args)
 
             // MainMenuManager implements the IUserInterfaceManager interface
             IUserInterfaceManager ui = new MainMenuManager();
+            bool justEnteredConsole = true;
```
[the remark this answers is not in the document]
Comment (karlarn) [reply TabloidCLI/Program.cs:15]: The Great Josh hath spoken. His word is as true and good as gold. This shall be implemented!

[the remark this answers is not in the document]
Comment (joshdbarton) [reply TabloidCLI/Program.cs]: 🤣

**Review by joshdbarton — APPROVED** [review]
🌈

**Pull request merged**